pkg/spoke/spokeagent.go

```diff
@@ -43,6 +46,7 @@ func (o *WorkloadAgentOptions) AddFlags(cmd *cobra.Command) {
 	flags := cmd.Flags()
 	// This command only supports reading from config
 	flags.StringVar(&o.HubKubeconfigFile, "hub-kubeconfig", o.HubKubeconfigFile, "Location of kubeconfig file to connect to hub cluster.")
+	flags.StringVar(&o.SpokeKubeconfigFile, "spoke-kubeconfig", o.SpokeKubeconfigFile, "Location of kubeconfig file to connect to spoke cluster.")
```

Comment:
can't we use the built-in --kubeconfig flag here?

Reply:
I  think it is better not, the built-in `--kubeconfig` in our case represents the cluster where the work-agent pod run.

Reply:
this should be explicit callout that if this is not set, but --kubeconfig is set. kubeconfig will be used to connect to spoke.